src/table/header-cell/index.tsx

```diff
@@ -84,26 +85,31 @@ export function TableHeaderCell<ItemType>({
 
   const headerId = useUniqueId('table-header-');
 
+  const clickableHeaderRef = useRef<HTMLDivElement>(null);
+  const { tabIndex: clickableHeaderTabIndex } = useSingleTabStopNavigation(clickableHeaderRef, { tabIndex });
+
   return (
     <TableThElement
       className={className}
       style={style}
       cellRef={cellRef}
       sortingStatus={sortingStatus}
       sortingDisabled={sortingDisabled}
+      focusedComponent={focusedComponent}
       hidden={hidden}
       colIndex={colIndex}
       columnId={columnId}
       stickyState={stickyState}
       tableRole={tableRole}
     >
       <div
+        ref={clickableHeaderRef}
         data-focus-id={`sorting-control-${String(columnId)}`}
         className={clsx(styles['header-cell-content'], {
           [styles['header-cell-fake-focus']]: focusedComponent === `sorting-control-${String(columnId)}`,
         })}
         aria-label={
-          column.ariaLabel
+          column.ariaLabel && !sortingDisabled
```

Comment:
Why the change in this condition?

Reply:
Oh that was actually not intended - will revert 👍